.travis.yml

```diff
@@ -20,7 +33,10 @@ before_install:
   - cd core
   - git submodule init
   - git submodule update
-  - ./occ maintenance:install --admin-user admin --admin-pass admin --database $DB
+  - mysql -u root -e "CREATE DATABASE oc_autotest; CREATE USER 'oc_autotest'@'localhost' IDENTIFIED BY 'oc_autotest'; GRANT ALL ON oc_autotest.* TO 'oc_autotest'@'localhost';"
```

Comment:
Don't do it yourself?
https://github.com/owncloud/activity/blob/master/.travis.yml#L20-L22 ;)